### ISSUE TITLE
UE 5.51: Unable to find sessions

### ISSUE THREAD

**Issue by 77Delta** [mordentral/AdvancedSessionsPlugin#100]
Hello! So recently I updated my project from 5.4.4 to 5.5.1 along with the AdvancedSteamSessions plugin. In this version, I can create sessions but i can’t find them.
![Captura de pantalla 2024-12-30 182319](https://github.com/user-attachments/assets/09a4e4ce-4c75-4fd2-b4e6-e2c2df0498b2)
The “On Failure” event doesnt execute, it only gets stuck searching sessions.

btw, here’s my DefaultEngine.ini:
```
[/Script/Engine.GameEngine]
+NetDriverDefinitions=(DefName="GameNetDriver",DriverClassName="OnlineSubsystemSteam.SteamNetDriver",DriverClassNameFallback="OnlineSubsystemUtils.IpNetDriver")

[OnlineSubsystem]
DefaultPlatformService=Steam
bHasVoiceEnabled=true

[Voice]
bEnabled=true

[OnlineSubsystemSteam]
bEnabled=true
SteamDevAppId=480
bInitServerOnClient=true
bUsesPresence=true
bUseLobbiesIfAvailable=true
bUseSteamNetworking=true

[/Script/OnlineSubsystemSteam.SteamNetDriver]
NetConnectionClassName="OnlineSubsystemSteam.SteamNetConnection"
```

**Comment by ickbird** [comment]
I am also having issues with 5.51, for 3 days been trying to connect with just the basic 3rd person, and simple host and join code.

**Comment by mordentral** [comment]
Did you update to the latest binaries for 5.5 or are you using 5.4's / the very first binaries for 5.5. I have to fake session results in 5.5 because epic is improperly formatting sessions results now.

Though it was making sessions unjoinable not unfindable, not finding sessions would be on the subsystems end. The BP nodes just bind to the search finished event to fire off.

**Comment by 77Delta** [comment]
I have the very latest version, i even reinstalled it right now, and the same happens.

**Comment by mordentral** [comment]
does the steam overlay show up for you?

**Comment by zeroy99** [comment]
> does the steam overlay show up for you?

Just something to keep in mind for Steam overlay, if you start your game with "-log -newconsole" it may disable steam overlay! We had this on our game.

**Comment by 77Delta** [comment]
> does the steam overlay show up for you?

Yes it does. Also, when i open the game in standalone the find session works but it shows this session:
![image](https://github.com/user-attachments/assets/8996b67e-f8b9-448d-8fe1-a5dc94756ff8)

**Comment by mordentral** [comment]
Without steam sockets the invalid MS is normal, unsure why there is no game name but that would be on the setup end of things. If that is an old session that is sticking around you need to be sure you destroy session when exiting.

**Comment by 77Delta** [comment]
I had steam sockets on since before
![image](https://github.com/user-attachments/assets/561eb998-f131-4cbd-aca1-91eb7e315461)
Also i already destroy the sessions when exiting.

**Comment by mordentral** [comment]
Not sure what to tell you without getting a direct debug instance on your session result, but the issues is very likely not in AS

**Comment by mordentral** [comment]
Yes, the project and the plugin, that is an engine change.

**Comment by 77Delta** [comment]
I see. What should I do then?

**Comment by mordentral** [comment]
load up the log in a development build and see what it says

**Comment by zekelty** [comment]
Hi, I had the same problem with the 480 app, I just entered my game ID, and everything worked out (which I got in the steam feed)

**Comment by nathanbowley98** [comment]
> Hi, I had the same problem with the 480 app, I just entered my game ID, and everything worked out (which I got in the steam feed)

I just doubled checked my DefaultEngine.ini and im still getting the issue even though I have my SteamDevAppId entered in, i have the steam feed but like it just wont even find matches now, its also giving me Failure Received, then connection lost errors on the client the moment the server makes a lobby even though the client hasn't started looking for the lobbies yet, which makes no sense to me

![Image](https://github.com/user-attachments/assets/3fed4e8b-791f-424d-a10b-5f47f53e16b0)

**Comment by nathanbowley98** [comment]
oh and im not necroing the thread, im just mentioning I have the exact same error from 5.4.4 -> 5.5.1 with sessions not being found at all, but I did find that out and I don't know if its exclusive to my game or if its also happening to others. This error corresponds with it happening 

![Image](https://github.com/user-attachments/assets/0742aa47-86fd-459e-b474-2201b907f4dc)

Then when I go to search for the lobbies it just comes back empty (unless you host the lobby as lan and then search for the lobby as lan than the lobby will show up)

![Image](https://github.com/user-attachments/assets/5cf2850a-be03-4b83-9245-3d253e84bb47)

**Comment by mordentral** [comment]
That log is showing the active subsystem being the lan one, you can't even load the steam subsystem in a PIE context.
